src/App.css

```diff
@@ -1,38 +1,233 @@
-.App {
-  text-align: center;
+@import url("https://fonts.googleapis.com/css2?family=Knewave&display=swap");
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ul,
+ol,
+li {
```

Comment:
As mentioned earlier, I can approve this now, but let's look for more scalable and consistent styling solutions. We'll need to refactor this vanilla CSS to another UI library- top ones to consider may be tailwind, material UI, chakra UI, and mantine